diff --git a/index.js b/index.js
index 8dbcf7e0a293c1fda7621e07b66568ec7cab01f9..ab40d7dafa7d23faf76298f2f3c328245cf320ec 100644
--- a/index.js
+++ b/index.js
@@ -74,10 +74,9 @@ module.exports = function withPreact(nextConfig = {}) {
           const itemsToPrepend =
             isServer && requiresRTSDiffHookPatch
               ? [
-                  'preact/debug',
                   'next-plugin-preact/patches/rts-invoke-diff-hook.js'
                 ]
-              : ['preact/debug'];
+              : [];
 
           if (isServer && requiresRTSDiffHookPatch) {
             process.stdout.write(